app/models/user.rb

```diff
@@ -0,0 +1,8 @@
+class User < ApplicationRecord
+
+  has_secure_password
+  validates :name, presence: true
+  validates :email, presence: true
+  validates :password, presence: true
```

Comment:
can we do it in single line?

Reply:
yes we can do like this -  
 validates :name,:email,:password, presence: true